source/04_sdk/02_reference/01_client_module/07_contribution/index.rst

```diff
@@ -0,0 +1,154 @@
+Contribution Endpoint
+---------------------
+
+.. warning:: WIP
+
+
+The ``DefaultRestClient`` includes a ``DefaultRestContributionEndpoint`` 
+with the following functionalities.
+
+
+Save contribution
+^^^^^^^^^^^^^^^^^
+
+Saving a contribution is done using the ``saveContribution`` method. The idea is that the client already works with an instance of the contribution. Client currently supports only composition inside contribution with ``create/update/delete`` change types. The creation of ``ContributionCreateDto`` explained below.
+
+    A very simplified example of saving contribution:
+
+.. code-block:: java
+
+    ContributionCreateDto contribution = [..]
+
+    // Contribution Saved
+    VersionUid versionUid = openEhrClient.contributionEndpoint(ehr).saveContribution(contribution);
+
+Create contribution using ContributionBuilder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A ``ContributionBuilder`` is a class that provides a new contribution instance. Key intentions of the ``ContributionBuilder`` is to hide much of the complexity and make the contribution easy to create. To start working with a ``ContributionBuilder`` we need an instance of the composition and audit details.
```

Comment:
The first sentence could be dropped, and "Key intentions" could be replaced with "Key objectives … are".
"is needed" instead of "we need"